packages/aws-cdk/lib/api/bootstrap/bootstrap-template.json

```diff
@@ -273,13 +273,20 @@
             },
             {
               "Action": [
-                "ecr:PutImage",        "ecr:InitiateLayerUpload",
-                "ecr:UploadLayerPart", "ecr:CompleteLayerUpload"
+                "ecr:PutImage",             "ecr:InitiateLayerUpload",
+                "ecr:UploadLayerPart",      "ecr:CompleteLayerUpload",
```

Comment:
any reason we have two per line in some places, and 1 per line in other places?

Reply:
Adam started this and I think it is to group related permissions together.